app/controllers/idv/verify_info_controller.rb

```diff
@@ -42,7 +42,7 @@ def self.step_info
       Idv::StepInfo.new(
         key: :verify_info,
         controller: self,
-        next_steps: [:phone],
+        next_steps: [:phone, :request_letter],
```

Comment:
Since you don't have to submit the phone step to get to RequestLetter, for FlowPolicy purposes it follows verify_info.